### DISCUSSION TITLE
How do I debug Aspire in vscode?

### DISCUSSION THREAD

**Discussion by NikiforovAll** [microsoft/aspire#4537] [Q&A]
I've tried to use "Default Debug Configuration" but nothing happens, the application is stalled. Is this scenario supported/intended?

I see:

```text
fail: Aspire.Hosting.Dcp.dcpctrl.ExecutableReconciler[0]
      failed to start Executable	{"Executable": {"name":"migrator-ak2gcbg"}, "Reconciliation": 253, "error": "run session could not be started: failed to connect to IDE run session notification endpoint: unexpected EOF"}
```

Additionally, Is it possible to put a breakpoint in services other than AppHost?

**Comment by davidfowl** [comment]
Are you using the latest version of the C# DevKit?

**Reply by NikiforovAll** [reply]
It is `v1.7.27` with a valid VS subscription associated with it

**Reply by davidfowl** [reply]
Can you reproduce this error when you create a new aspire starter project and run?
